spec/support/database_cleaner.rb

```diff
@@ -0,0 +1,21 @@
+RSpec.configure do |config|
+  config.before(:suite) do
+    DatabaseCleaner.clean_with(:truncation)
```

Comment:
зачем здесь truncation?

Reply:
Честно не знаю. Я читал https://thoughtbot.com/blog/how-we-test-rails-applications. Там говорилось.

This is where Database Cleaner comes in. Database Cleaner allows you to configure when each strategy is used. I recommend reading Avdi’s post for all the gory details. It’s a pretty painless setup, and I typically copy this file from project to project, or use Suspenders so that it’s set up out of the box. File: https://gist.github.com/jsteiner/8362013